cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt

```diff
@@ -0,0 +1,371 @@
+name "Google Committed Use Discount Recommender"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy finds Google Committed Use Discount Recommendations and reports when it finds them. See the [README](https://github.com/flexera/policy_templates/tree/master/cost/google/recommender) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```

Comment:
First link does not match the path of the readme - shouldn't this be `[README](https://github.com/flexera/policy_templates/tree/master/cost/google/cud_recommendations)` ?

Reply:
Second link is currently rightscale link, this should be replaced with `[docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more.` instead